db/migrate/20150611145301_create_roles.rb

```diff
@@ -1,9 +1,9 @@
 class CreateRoles < ActiveRecord::Migration
   def change
     create_table :roles do |t|
-      t.integer :title # %(:student :marker :instructor)
-      t.integer :course_id
-      t.integer :user_id
+      t.integer :title, default: :student # %(:student :marker :instructor), default a student
+      t.belongs_to :course # generate foreign key course_id
```

Comment:
Might want to add:

    null: false

Double protection, in case something happened during validation